core/src/main/scala/com/nvidia/spark/rapids/tool/planparser/SQLPlanParser.scala

```diff
@@ -125,11 +125,13 @@ case class ExecInfo(
   }
 
   private def getOpAction: OpActions.OpAction = {
-    if (shouldIgnore) {
-      OpActions.IgnorePerf
-    } else if (shouldRemove) {
+    // shouldRemove is checked first because sometimes an exec could have both flag set to true,
```

Comment:
This is a quick fix for a discrepancy I found in the reporting. It had no affect on the accuracy of the results.
Some execs might have both flags set to True. In that case, "NoPerf" should take precedence.